index.html

```diff
@@ -46,30 +46,29 @@ <h4 id="stocksTitle" class= "boxTitle">Stocks Heading Test</h4>
           </span>
         </div>
 
-        <!-- stocks results list -->
-        <div class="row">
-          <div class="row">
-            <div class="columns small-2">Symbol</div>
-            <div class="columns small-2">Open</div>
-            <div class="columns small-2">High</div>
-            <div class="columns small-2">low</div>
-            <div class="columns small-2">close</div>
-            <div class="columns small-2">volume</div>
-          </div>
-        </div>
-        <div class="row align-right" id="render-stock"></div>
-    
-        <!-- <ul id="stocksList">
-          <li>Test li, please delete before working on JS</li>
-          <li>Test li, please delete before working on JS</li>
-          <li>Test li, please delete before working on JS</li>
-          <li>Test li, please delete before working on JS</li>
-          <li>Test li, please delete before working on JS</li>
-        </ul> -->
-          <div class="row">
-            <button id="clearBtnNews" class="clearBtn button" type="button">Clear</button>
-          </div>
-        </div>
+        <!-- stocks results table -->
+        <table>
+          <thead>
+            <tr>
+              <th>Symbol</th>
+              <th>Open</th>
+              <th>High</th>
+              <th>Low</th>
+              <th>Close</th>
+              <th>Volume</th>
+              <!-- up/down arrow example -->
+              <th><i class="fas fa-level-up-alt"></i></th>
```

Comment:
these headings should be empty - the icons go in the stock rows
```suggestion
              <th></th>
```